keras_retinanet/models/retinanet.py

```diff
@@ -102,6 +128,17 @@ def default_regression_model(num_anchors, pyramid_feature_size=256, regression_f
 
 
 def __create_pyramid_features(C3, C4, C5, feature_size=256):
+    """ Creates the FPN layers on top of the backbone features.
+
+    Args
+        C3           : Feature stage C3 from the backbone.
+        C4           : Feature stage C4 from the backbone.
+        C5           : Feature stage C5 from the backbone.
+        feature_size : The feature size to use for the resulting feature levels.
+
+    Returns
+        A list of feature levels (P3, P4, P5, P6, P7).
```

Comment:
Perhaps change the comment to use list notation rather than tuple notation, just in case this could be misconstrued.

Reply:
:+1: